plugins/automox/icon_automox/actions/execute_vulnerability_sync_actions/action.py

```diff
@@ -0,0 +1,49 @@
+import insightconnect_plugin_runtime
+from .schema import ExecuteVulnerabilitySyncActionsInput, ExecuteVulnerabilitySyncActionsOutput, Input, Output, \
+    Component
+
+
+# Custom imports below
+
+
+class ExecuteVulnerabilitySyncActions(insightconnect_plugin_runtime.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name='execute_vulnerability_sync_actions',
+            description=Component.DESCRIPTION,
+            input=ExecuteVulnerabilitySyncActionsInput(),
+            output=ExecuteVulnerabilitySyncActionsOutput())
+
+    def run(self, params={}):
+        org_id = params.get(Input.ORG_ID)
+        action_set_id = params.get(Input.ACTION_SET_ID)
+        actions = params.get(Input.ACTIONS)
+        if actions is None or len(actions) == 0:
+            raise insightconnect_plugin_runtime.exceptions.PluginException(
+                cause="Invalid input",
+                assistance="Actions cannot be empty."
+            )
```

Comment:
If actions is marked as `required: true` in the `plugin.spec.yaml`, then will this code block ever be hit?

Reply:
In my testing I found that an empty array could be sent even though actions was marked as required, so I added this to handle that situation